src/pyscheduling/Problem.py

```diff
@@ -4,14 +4,19 @@
 from enum import Enum
 from pathlib import Path
 from time import perf_counter
+import numpy as np
```

Comment:
Akram, I think we should remove the dependency to numpy, we're only using it for random number generation in the case of normal distribution. 

The dependency is currently respected because we have matplotlib which also depends on numpy but if we change the plotting to plotly it will cause a problem. 

We can implement the normal dist generation using the random lib